### PR TITLE
feat(Python): Added documenation for Cloud Resource Context

### PR DESCRIPTION
**Pull request by antonpirker**
Information that is added by the new `CloudContextIntegration` when the app of the user is run in a cloud provider like AWS or Google Cloud Platform.

### CODE REVIEW

**Comment by antonpirker** [comment]
Related to this develop documentation update: https://github.com/getsentry/develop/pull/825

File: src/platforms/python/common/configuration/integrations/cloudcontext.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+When the SDK starts up information from the cloud provider the app is running in is retrieved and set in all the error and performance events sent to Sentry.
+The [developer documentation](https://develop.sentry.dev/sdk/event-payloads/contexts/#runtime-context) list all the information that is added.
+
+In AWS EC2 the context looks like this:
+
+![AWS EC2 Cloud Resource Context in Sentry event](cloud-context-aws-ec2.png)
+
+In Google Cloud Platform GCE the context looks like this:
+
+![Google GCE Cloud Resource Context in Sentry event](cloud-context-gcp-gce.png)
```
Comment (lizokm) [inline src/platforms/python/common/configuration/integrations/cloudcontext.mdx:42]:
```suggestion
## Behavior

When the SDK starts up, information from the cloud provider the app is running in is retrieved and added to all error and performance events sent to Sentry.
The [developer documentation](https://develop.sentry.dev/sdk/event-payloads/contexts/#runtime-context) lists all the information that's being added.

In AWS EC2 the context looks like this:

![AWS EC2 Cloud Resource Context in Sentry event](cloud-context-aws-ec2.png)

In Google Cloud Platform GCE the context looks like this:

![Google GCE Cloud Resource Context in Sentry event](cloud-context-gcp-gce.png)
```

**Review by lizokm — APPROVED** [review]
Just a few small language tweaks, but otherwise this looks good!

**Comment by antonpirker** [comment]
I will merge this once the feature is released

**Pull request merged**